### ISSUE TITLE
Can't use "Sweet Dreams" Achivement

### ISSUE THREAD

**Issue by Spydrouge** [Spydrouge/pixelmon-ghosts-16#13]
Because Minecraft Achievements have to be unlocked in order, you can't unlock the Sweet Dreams achievement until you've gotten the correct pickaxes, and done other things on the advancements list. Quests can artificially complete those advancements, which  might help, but just holding a bed is better (if less immersive)

Also: Auto-claim the "Arceus blesses you with a vision" or the quest after it makes no sense.

Also: Tell the player to make a pickaxe head so they can drop Geodude to their doom

Also: Auto-claim "And Arceus Saw that it was Good"

**Comment by Spydrouge** [comment]
Fix Attempted: We give the player the root adventure quest just before this quest. Fix Untested

**Comment by Spydrouge** [comment]
This never worked. 

We decided to test a bunch of scenarios. Sometimes, we joined a brand new world, slept immediately, and immediately got the sweet dreams achievement without even needing the root adventure quest. Other times we joined a new game and no amount of sleeping ever gave us the achievement.

We were unable to figure out what exactly caused the achivement to proc. It was extremely inconsistent. For this reason, I begrudgingly changed the quest to only require you to hold a bed in your hand.
